vllm/worker/xpu_worker.py

```diff
@@ -199,7 +203,7 @@ def init_worker_distributed_environment(self) -> None:
         ensure_model_parallel_initialized(
             parallel_config.tensor_parallel_size,
             parallel_config.pipeline_parallel_size)
-
+        torch.distributed.all_reduce(torch.zeros(1).xpu())
```

Comment:
why need this, please add some comments

Reply:
We still need a global all_reduce even TP not enabled for the oneCCL issue.